unit_test/test_crypt/rsa_verify.c

```diff
@@ -103,6 +103,9 @@ uint8_t m_libspdm_rsa_pkcs1_signature[] = {
 /* Default public key 0x10001. */
 uint8_t m_libspdm_default_public_key[] = { 0x01, 0x00, 0x01 };
 
+/* input public key 0x03. */
+uint8_t m_libspdm_rsa_public_exponent[] = { 0x03 };
```

Comment:
rename to `m_libspdm_**test**_rsa_public_exponent`

Reply:
Thanks. I have fixed it.